server/genquery2/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+find_package(FLEX 2.6.1 REQUIRED)
+find_package(BISON 3.0.4 REQUIRED)
+
+set(IRODS_FLEX_LEXER_NAME irods_lexer)
+set(IRODS_BISON_PARSER_NAME irods_parser)
+
+# Normally, we would do this at build-time, but FLEX_TARGET and BISON_TARGET don't actually define targets,
+# therefore there's no good mechanism to add a dependency to the flex and bison targets.
+# The source file property OBJECT_DEPENDS is *supposed* to work for this, but doesn't.
```

Comment:
I don't know that an issue is needed. The flex/bison cmake mechanism is outside our control.

Unless the flex/bison cmake tools grow additional features, I don't think there's anything for us to do.

Thoughts?

Reply:
I was under the impression that this is a CMake bug (that is, an issue in CMake the tool, not *our* CMake), so that's why I was wondering. We've referenced open issues on other projects elsewhere as explanations for why we do certain things that may seem strange

Don't need to put any effort into finding it - this was mostly out of curiosity.

Reply:
Wait, I missed that last part of the comment on L9.

@SwooshyCueb Do we need an issue for this?

Reply:
Yeah, it might be a good idea to open an issue to investigate this further.

Reply:
Cool. Marking as resolved.